cli/commands/terraform/action.go

```diff
@@ -832,3 +845,7 @@ func setTerragruntNullValues(terragruntOptions *options.TerragruntOptions, terra
 
 	return varFile, nil
 }
+
+func useLegacyNullValues() bool {
+	return os.Getenv(useLegacyNullValuesEnvVar) != ""
```

Comment:
Let's check that it's set to `"1"` instead of anything other than `""`.

That way people don't have to unset the env var if they want it dynamically set.